### PR TITLE
enhance FFTW easyblock to support SVE CPU feature and building with Fujitsu compiler

### PR DESCRIPTION
**Pull request by migueldiascosta**
requires ~https://github.com/easybuilders/easybuild-framework/pull/3677~

being tested at Fugaku

### CODE REVIEW

File: easybuild/easyblocks/f/fftw.py [hunk 1]
```diff
@@ -187,9 +188,14 @@ def run_all_steps(self, *args, **kwargs):
...
-                # NEON (ARM) only for single precision and double precision (on AARCH64)
-                if (prec == 'single' and (self.asimd or self.neon)) or (prec == 'double' and self.asimd):
-                    prec_configopts.append('--enable-neon')
+                if self.sve:
+                    # SVE (ARM) only for single precision and double precision (on AARCH64 if sve feature is present)
+                    if prec == 'single' or prec == 'double':
+                        prec_configopts.append('--enable-sve --enable-armv8-cntvct-el0')
+                elif self.asimd or self.neon:
+                    # NEON (ARM) only for single precision and double precision (on AARCH64)
+                    if prec == 'single' or prec == 'double':
```
Comment (boegel) [inline easybuild/easyblocks/f/fftw.py:197]: `double` should only be enabled if `asimd` is supported?

```python
if prec == 'single' or (prec == 'double' and self.asimd`):

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyblocks/f/fftw.py (hunk 1 above)
Comment (migueldiascosta) [reply easybuild/easyblocks/f/fftw.py]: fixed via force push

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _FFTW-3.3.8-gompi-2020b.eb_ 

Build succeeded for 1 out of 1 (1 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/9c04bf61455864a20fc34392effa111a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _FFTW-3.3.8-gompi-2019b.eb_ 
 * **SUCCESS** _FFTW-3.3.9-gompi-2021.04.eb_ 
 * **SUCCESS** _FFTW-3.3.8-intel-2020b.eb_ 

Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3595.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/b596a1018e271fbd287302bffa16714e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _FFTW-3.3.8-gompi-2020b.eb_ 

Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild2.novalocal - Linux centos linux 8.3.2011, POWER, IBM pSeries (emulated by qemu) (power9le), Python 3.6.8
See https://gist.github.com/472ccbd1c99448232746f192c30eda5d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _FFTW-3.3.8-gompi-2020b.eb_ 

Build succeeded for 1 out of 1 (1 easyconfigs in total)
gw4a64fxlogin00 - Linux RHEL 8.2, AArch64, UNKNOWN (a64fx), Python 3.6.8
See https://gist.github.com/d1173597cc12cff16543c3788f046f2c for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**